<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSGTools Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
     <property name="title">
      <string>Free Hand's Parameters</string>
     </property>
     <property name="saveCollapsedState">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Tolerance</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsDoubleSpinBox" name="toleranceQgsDoubleSpinBox">
        <property name="decimals">
         <number>10</number>
        </property>
        <property name="maximum">
         <double>100000000000000000.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Smooth Iterations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsSpinBox" name="smoothIterationsQgsSpinBox">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Smooth offset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsDoubleSpinBox" name="smoothOffsetQgsDoubleSpinBox">
        <property name="decimals">
         <number>10</number>
        </property>
        <property name="maximum">
         <double>100000000000000004764729344.000000000000000</double>
        </property>
        <property name="value">
         <double>0.250000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Algorithm Iterations</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QgsSpinBox" name="algIterationsQgsSpinBox">
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Number of points removed on undo action</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QgsSpinBox" name="undoQgsSpinBox">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Final Simplify Tolerance</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QgsDoubleSpinBox" name="finalToleranceQgsDoubleSpinBox">
        <property name="decimals">
         <number>10</number>
        </property>
        <property name="maximum">
         <double>100000000000000000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.300000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox_2">
     <property name="title">
      <string>Generic Selector's Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="addParameterLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="addPushButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources.qrc">
            <normaloff>:/plugins/DsgTools/icons/plus.png</normaloff>:/plugins/DsgTools/icons/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removePushButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../resources.qrc">
            <normaloff>:/plugins/DsgTools/icons/remove.png</normaloff>:/plugins/DsgTools/icons/remove.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="blackListWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <item>
         <property name="text">
          <string>moldura</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox_4">
     <property name="title">
      <string>Right Angle Tool Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Minimum Segment Distance (meters)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsSpinBox" name="minSegmentDistanceQgsSpinBox">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox_3">
     <property name="title">
      <string>Raster Info Tool's Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Value setter tool precision (decimal points)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsSpinBox" name="decimalQgsSpinBox">
        <property name="maximum">
         <number>21</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox_5">
     <property name="title">
      <string>Validation Toolbar Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QComboBox" name="modelPathComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Models' reading directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBeforeRunModelCheckBox">
        <property name="text">
         <string>Ask for user confirmation before running models</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="resetModelsCheckBox">
        <property name="text">
         <string>Remove added models when exiting the application</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="removeModelsProjectCheckBox">
        <property name="text">
         <string>Remove added models when swtiching QGIS project</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="addModelPathPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Add a custom path to look for QGIS Processing models.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources.qrc">
          <normaloff>:/plugins/DsgTools/icons/plus.png</normaloff>:/plugins/DsgTools/icons/plus.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="loadModelOutputCheckBox">
        <property name="text">
         <string>Automatically load output layers to canvas</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
